src/OpenTelemetry.Instrumentation.StackExchangeRedis/Implementation/RedisMetrics.cs

```diff
@@ -0,0 +1,41 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+using System.Diagnostics.Metrics;
+using System.Reflection;
+using OpenTelemetry.Internal;
+
+namespace OpenTelemetry.Instrumentation.StackExchangeRedis.Implementation;
+
+internal class RedisMetrics : IDisposable
+{
+    internal const string DurationMetricName = "db.client.operation.duration";
+
+    internal static readonly Assembly Assembly = typeof(StackExchangeRedisInstrumentation).Assembly;
+    internal static readonly AssemblyName AssemblyName = Assembly.GetName();
+    internal static readonly string InstrumentationName = AssemblyName.Name!;
+    internal static readonly string InstrumentationVersion = Assembly.GetPackageVersion();
+
+    private readonly Meter meter;
+
+    public RedisMetrics()
+    {
+        this.meter = new Meter(InstrumentationName, InstrumentationVersion);
+
+        this.DurationHistogram = this.meter.CreateHistogram<double>(
+            DurationMetricName,
+            unit: "s",
+            description: "Total client request duration, including processing, queue and server duration.");
```

Comment:
```suggestion
            description: "Duration of database client operations.");
```

ref: https://github.com/open-telemetry/semantic-conventions/blob/v1.27.0/docs/database/database-metrics.md?plain=1#L82